keyboards/novelkeys/nk_plus/info.json

```diff
@@ -0,0 +1,196 @@
+            { "flags": 1, "matrix": [2, 0], "x": 0, "y": 32 },
+            { "flags": 1, "matrix": [2, 1], "x": 13, "y": 32 },
+            { "flags": 1, "matrix": [2, 2], "x": 37, "y": 32 },
+            { "flags": 4, "matrix": [2, 3], "x": 54, "y": 32 },
+            { "flags": 4, "matrix": [2, 4], "x": 67, "y": 32 },
+            { "flags": 4, "matrix": [2, 5], "x": 80, "y": 32 },
+            { "flags": 4, "matrix": [2, 6], "x": 93, "y": 32 },
+            { "flags": 4, "matrix": [2, 7], "x": 106, "y": 32 },
+            { "flags": 4, "matrix": [2, 8], "x": 118, "y": 32 },
+            { "flags": 4, "matrix": [2, 9], "x": 131, "y": 32 },
+            { "flags": 4, "matrix": [2, 10], "x": 144, "y": 32 },
+            { "flags": 4, "matrix": [2, 11], "x": 157, "y": 32 },
+            { "flags": 4, "matrix": [2, 12], "x": 170, "y": 32 },
+            { "flags": 4, "matrix": [2, 13], "x": 182, "y": 32 },
+            { "flags": 1, "matrix": [2, 15], "x": 203, "y": 32 },
+            { "flags": 1, "matrix": [2, 16], "x": 224, "y": 32 },
+            { "flags": 1, "matrix": [3, 16], "x": 224, "y": 48 },
+            { "flags": 1, "matrix": [3, 15], "x": 211, "y": 48 },
+            { "flags": 1, "matrix": [3, 14], "x": 194, "y": 48 },
+            { "flags": 4, "matrix": [3, 13], "x": 176, "y": 48 },
+            { "flags": 4, "matrix": [3, 12], "x": 163, "y": 48 },
+            { "flags": 4, "matrix": [3, 11], "x": 150, "y": 48 },
+            { "flags": 4, "matrix": [3, 10], "x": 138, "y": 48 },
+            { "flags": 4, "matrix": [3, 9], "x": 125, "y": 48 },
+            { "flags": 4, "matrix": [3, 8], "x": 112, "y": 48 },
+            { "flags": 4, "matrix": [3, 7], "x": 99, "y": 48 },
+            { "flags": 4, "matrix": [3, 6], "x": 86, "y": 48 },
+            { "flags": 4, "matrix": [3, 5], "x": 74, "y": 48 },
+            { "flags": 4, "matrix": [3, 4], "x": 61, "y": 48 },
+            { "flags": 1, "matrix": [3, 2], "x": 40, "y": 48 },
+            { "flags": 1, "matrix": [3, 1], "x": 13, "y": 48 },
+            { "flags": 1, "matrix": [3, 0], "x": 0, "y": 48 },
+            { "flags": 1, "matrix": [4, 0], "x": 0, "y": 64 },
+            { "flags": 1, "matrix": [4, 1], "x": 13, "y": 64 },
+            { "flags": 1, "matrix": [4, 2], "x": 35, "y": 64 },
+            { "flags": 1, "matrix": [4, 3], "x": 51, "y": 64 },
+            { "flags": 1, "matrix": [4, 4], "x": 67, "y": 64 },
+            { "flags": 4, "matrix": [4, 8], "x": 122, "y": 64 },
+            { "flags": 1, "matrix": [4, 13], "x": 176, "y": 64 },
+            { "flags": 1, "matrix": [4, 14], "x": 198, "y": 64 },
+            { "flags": 1, "matrix": [4, 15], "x": 211, "y": 64 },
+            { "flags": 1, "matrix": [4, 16], "x": 224, "y": 64 }
+        ],
+        "max_brightness": 120
+    },
+    "rgblight": {
+        "pin": "B4"
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                { "matrix": [0, 0], "w": 1, "x": 0, "y": 0 },
```

Comment:
```suggestion
                { "matrix": [0, 0], "x": 0, "y": 0 },
```
etc.

Reply:
...

Reply:
This has now been fixed. sorry for that.
I got a bit confused before the qmk cmd command gave the matrix output with widths:)